### PR TITLE
Event filter fix

### PR DESCRIPTION
**Pull request by MohamedFarouk-HMCTS**
This is to fix the issue with no schema validations of command handler due to single war class loading EventFilter of Event Listener for handlers which then skips validation due to no match on the names

### CODE REVIEW

File: generators/messaging-adapter-generator/src/main/java/uk/gov/justice/raml/jms/core/JmsEndpointGenerator.java [hunk 1]
```diff
@@ -79,11 +79,11 @@ public void run(final Raml raml, final GeneratorConfig configuration) {
     therefore we need all messages with consecutive numbers there. Messages need to be in correct order in order to update the view correctly.
     */
 
-    private boolean shouldGenerateEventFilter(final Resource resource, final MessagingAdapterBaseUri baseUri) {
+    static boolean shouldGenerateEventFilter(final Resource resource, final MessagingAdapterBaseUri baseUri) {
```
Comment (mapingo) [inline generators/messaging-adapter-generator/src/main/java/uk/gov/justice/raml/jms/core/JmsEndpointGenerator.java]: If we are making these two methods static and accessible from another class, then shouldn't they be in another Class?

File: generators/messaging-adapter-generator/src/main/java/uk/gov/justice/raml/jms/core/JmsEndpointGenerator.java [hunk 2]
```diff
@@ -79,11 +79,11 @@ public void run(final Raml raml, final GeneratorConfig configuration) {
...
         return EVENT_LISTENER.equals(baseUri.component()) && !containsGeneralJsonMimeType(resource.getActions());
     }
 
-    private boolean shouldListenToAllMessages(final Resource resource, final MessagingAdapterBaseUri baseUri) {
+    static boolean shouldListenToAllMessages(final Resource resource, final MessagingAdapterBaseUri baseUri) {
```
Comment (mapingo) [inline generators/messaging-adapter-generator/src/main/java/uk/gov/justice/raml/jms/core/JmsEndpointGenerator.java]: As above

Comment (MohamedFarouk-HMCTS) [reply generators/messaging-adapter-generator/src/main/java/uk/gov/justice/raml/jms/core/JmsEndpointGenerator.java]: Moved to a separate class JmsEndPointGeneratorUtil

File: generators/messaging-adapter-generator/src/main/java/uk/gov/justice/raml/jms/core/JmsEndpointGenerator.java (hunk 1 above)
Comment (MohamedFarouk-HMCTS) [reply generators/messaging-adapter-generator/src/main/java/uk/gov/justice/raml/jms/core/JmsEndpointGenerator.java]: Moved to a separate class JmsEndPointGeneratorUtil

**Pull request merged**